examples/webgpu_compute_particles_snow.html

```diff
@@ -14,11 +14,11 @@
 		<script type="importmap">
 			{
 				"imports": {
-					"three": "../build/three.webgpu.js",
-					"three/webgpu": "../build/three.webgpu.js",
+					"three": "../src/Three.WebGPU.js",
+					"three/webgpu": "../src/Three.WebGPU.js",
```

Comment:
These break in `gh-pages`:
[webgpu_compute_particles_snow](https://threejs.org/examples/?q=snow#webgpu_compute_particles_snow)

Reply:
#30432